### DISCUSSION TITLE
Feature request: Add Web document to Workspace Knowledge collection

### DISCUSSION THREAD

**Discussion by danpicton** [open-webui/open-webui#6118] [Ideas]
Currently, the following options are available when creating a Knowledge collection in the Workspace:
<img width="421" alt="image" src="https://github.com/user-attachments/assets/daf9a58e-7f09-4b2d-88df-556952db7879">

It would be useful to be able to add web-derived documents, similar to how you can parse a remote (web) document using the `#` prefix in the chat interface (e.g. `#https://docs.openwebui.com/tutorials/features/rag#document-parsing`).

I'd like to grab a bunch of links (or even a site map), dump them into my knowledge collection, and chat to them.

**Comment by arijit1201** [comment]
Has there been any development on this ?

**Comment by diskreet90** [comment]
Also wanting to see this and I will totally ditch AnythingLLM

**Comment by muntedcrocodile** [comment]
Would love to see this as a feature. Would make the working with a knowledge base in a research workflow far more convenient.

**Comment by Askath** [comment]
I wonder if there is a way to do this with tools?

**Comment by suleimanelkhoury** [comment]
@tjbck Any Info on this?

**Comment by bdd4329** [comment]
Any update?

**Comment by seppel123** [comment]
This would be a big feature, still no infos on this?

**Comment by Classic298** [comment]
Good idea, will put this on my todo list

**Comment by jdstu30** [comment]
Also thing this would be a very useful feature, being able to add multiple pages in bulk and store for later retrieval would be very useful.
